src/ngx_http_datadog_module.cpp

```diff
@@ -58,12 +56,12 @@ using namespace datadog::nginx;
 // There are two sets of places Datadog commands can appear: either "anywhere,"
 // or "anywhere but in the main section."  `anywhere` and `anywhere_but_main`
 // are respective shorthands.
+// Also, this definition of "anywhere" excludes "if" blocks. "if" blocks are an
+// abomination that nginx is now unable to kill.
```

Comment:
Not disagreeing, but either kinder wording or a link explaining the details might be a good idea.

Reply:
Yeah I'll tone it down.